addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/XxeScanRule.java

```diff
@@ -257,7 +257,7 @@ private void localFileReflectionAttack(HttpMessage msg) {
             // First replace the values in all the Elements by the Attack Entity
             String originalRequestBody = msg.getRequestBody().toString();
             String requestBody = createLfrPayload(originalRequestBody);
-            if (localFileReflectionTest(msg, requestBody, payload)) {
+            if (localFileReflectionTest(msg, requestBody)) {
```

Comment:
The corresponding variable should be removed (which means moving the catch into the method).